src/org/labkey/test/tests/SpecimenBaseTest.java

```diff
@@ -60,19 +63,28 @@ protected void setupRequestabilityRules()
         clickFolder(getFolderName());
         waitAndClick(Locator.linkWithText("Manage Study"));
         waitAndClick(Locator.linkWithText("Manage Requestability Rules"));
+
+        // Wait for grid to render
+        waitForElement(Locator.xpath("//table[contains(@class, 'x4-grid-table')]"));
+        Ext4GridRef gridRef = _ext4Helper.queryOne("panel[title='Active Rules']", Ext4GridRef.class);
+
+        // Expect three rows
+        int rowCount = gridRef.getRowCount();
+        log("How many rows? " + rowCount);
+        Assert.assertEquals(3, rowCount);
+
         // Verify that LOCKED_IN_REQUEST is the last rule
-        waitForElement(Locator.xpath("//div[contains(@class, 'x-grid3-row-last')]//div[text()='Locked In Request Check']"));
-        click(Locator.xpath("//div[contains(@class, 'x-grid3-row-last')]//div[text()='Locked In Request Check']"));
+        log(gridRef.getFieldValue(rowCount, "name").toString());
+        Assert.assertEquals("Locked In Request Check", gridRef.getFieldValue(rowCount, "name"));
```

Comment:
Assertions should have a message.
```suggestion
        Assert.assertEquals("Last requestability rule", "Locked In Request Check", gridRef.getFieldValue(rowCount, "name"));
```